src/index.ts

```diff
@@ -10,40 +21,24 @@ export = {
         'no-undef': 'off',
 
         // Rules replaced by @typescript-eslint versions:
-        camelcase: 'off',
-        indent: 'off',
-        'no-array-constructor': 'off',
-        'no-unused-vars': 'off',
+        ...Object.fromEntries(equivalents.map((name) => [name, 'off'])),
         'no-use-before-define': 'off',
-        'no-useless-constructor': 'off',
-        quotes: 'off',
```

Comment:
This block is replaced by mostly (except for one) programmatic generation of the turning off of the rules.